hq/test/aws/ec2/EC2Test.scala

```diff
@@ -140,6 +142,38 @@ class EC2Test extends FreeSpec with Matchers with Checkers with PropertyChecks
     }
   }
 
+  "VPC" - {
+    val sgs1 = SGOpenPortsDetail("Ok", "eu-west-1", "name-1", "id-122", "vpc-1", "tcp", "1099", "Yellow", false )
+    val sgs2 = SGOpenPortsDetail("Ok", "eu-west-2", "name-2", "id-122", "vpc-2", "tcp", "1099", "Yellow", false )
+    val sgs3 = SGOpenPortsDetail("Ok", "eu-west-2", "name-3", "id-122", "vpc-3", "tcp", "1099", "Yellow", false )
+
+    val vpcsMap = Map(
+      "vpc-1" -> new Vpc().withVpcId("vpc-1").withTags(new Tag("Name", "name-1")),
+      "vpc-2" -> new Vpc().withVpcId("vpc-2").withTags(new Tag("Name", "name-2")),
+      "vpc-3" -> new Vpc().withVpcId("vpc-3")
+    )
+    val sgsList = List(sgs1, sgs2, sgs3)
+
+    "get vpcs"  in {
+
+      def vpcsF (ec2: AmazonEC2Async, vpcIds : List[String]) : Attempt[Map[String, Vpc]] = {
```

Comment:
Since this is only used once would it be easier(/clearer) to just hard-code the result? I'm not sure - could go either way on it! (I'm just always nervous when tests look like they need tests)